src/backend/oracle-data-api/src/test/java/ca/bc/gov/open/jag/tco/oracledataapi/util/RandomUtil.java

```diff
@@ -299,9 +300,11 @@ public static String randomAlphanumeric(int count) {
 
 	/**
 	 * Returns a random date within the last year.
+	 * @throws ParseException
 	 */
-	public static Date randomDate() {
-		return DateUtils.ceiling(randomDateTime(), Calendar.DAY_OF_MONTH);
+	public static Date randomDate() throws ParseException {
+		// ceiling dateTime to top of day
+		return DateUtils.parseDate(DateFormatUtils.ISO_8601_EXTENDED_DATE_FORMAT.format(randomDateTime()), DateFormatUtils.ISO_8601_EXTENDED_DATE_FORMAT.getPattern());

```

Comment:
This was a fix for DateUtils.ceiling() ... that method doesn't work and moves the date back one day.